docs/src/manual/grid.md

```diff
@@ -168,26 +168,13 @@ Next, we define some helper functions that take care of the node handling.
 Ferrite.getnodes(grid::SmallGrid) = grid.nodes_test
 Ferrite.getnodes(grid::SmallGrid, v::Union{Int, Vector{Int}}) = grid.nodes_test[v]
 Ferrite.getnnodes(grid::SmallGrid) = length(grid.nodes_test)
+Ferrite.getnodenumbertype(::SmallGrid) = Float64
```

Comment:
Sorry if I misunderstand this one, but `getnodenumbertype` is technically the data type of the coordinate, right?

Reply:
Yes, perhaps it should be `getcoordinatenumbertype` instead? That's even harder to read though.

Reply:
Maybe just `coordinatetype` or `coordinate_type`. Not sure what the current convention about incuding `get` in a method name is.

Reply:
`coordinatetype` sounds like it should return `Vec{dim,T}` (or for the `SmallGrid` `NTuple{dim,T}`) though, while this only returns `T` - that was what made me choose the `numbertype` name.

The `get` was analogous to `getdim` and `getcelltype` but to be fair there really are no well-defined rules about prepending `get` or not in Ferrite.

`get_coordinate_number_type`? `coodinatenumbertype`?

Reply:
Is `get_coordinate_eltype` or `getcoordeltype` an idea?